### PR TITLE
0.3.3 - fixed html parsing, restored functionality

### PR DESCRIPTION
**Pull request by markmark1**
fixes bug please make 

In addition to your existing fixes, please bump the lib version references to 0.3.3 in the pull request. That way, I can publish it to pypi for you.

That is, change these references to 0.3.3

$ git grep 0.3.2
CHANGES.txt:v0.3.2, 2014-05-01 Update configparser lib version to 3.5.0b2
clippercard/__init__.py:__version__ = '0.3.2'
setup.py:    version='0.3.2',

### CODE REVIEW

**Comment by markmark1** [comment]
pls make new release on pypi

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-v0.3.2, 2014-05-01 Update configparser lib version to 3.5.0b2
```
Comment (goldengate88) [inline CHANGES.txt:1]: one more small fix please, you actually erased the changelog for the previous version 0.3.2, what I meant here was to:

1. add a new line on top describing the changes in your fix, something like: `v0.3.3, 2017-09-05 fixed html parsing, restored functionality`
2. preserve the previous line on 0.3.2

**Review by goldengate88 — CHANGES_REQUESTED** [review]
one more changelog request, and I'd be happy to merge and publish 0.3.3

**Comment by goldengate88** [comment]
Thank you @markmark1 for fixing this long standing and unfortunately ignored bug. We appreciate your contribution to this project.

**Comment by markmark1** [comment]
Ying s pls change it and merge and submit

________________________________

**Comment by goldengate88** [comment]
@markmark1 - I cannot change your source branch to update this PR because the source repo is yours. Ideally, all encompassing commits for a particular change should come in a consolidated PR instead of multiple people piecemealing it together. Generally it's up to the PR author (you) to submit follow-up until all review requests are fulfilled.

This time I will make an exception and merge this PR as is.

Thanks again and you're welcome to submit future changes.

**Review by goldengate88 — APPROVED** [review]

**Comment by goldengate88** [comment]
1. changelogs and docs updated in #2 
1. https://pypi.python.org/pypi/clippercard/0.3.3 should be available soon

**Pull request merged**